doc/UPGRADE.md

```diff
@@ -1,6 +1,28 @@
 Netgen Tags Bundle upgrade instructions
 =======================================
 
+Upgrade from 5.0 to 5.4
+-----------------------
+
+Tags Bundle 5.4 introduces support for tag visibility - now you can hide and reveal tags, just the same as content in 
+Ibexa Platform. Tags that are explicitly hidden or invisible because one of their ancestor is hidden cannot be added to 
+content.
+
+Please make sure to update Netgen Tags table to include `is_hidden` and `is_invisible` columns.
+
+If you're using MySQL, run the commands below:
+
+```sql
+ALTER TABLE `eztags` ADD COLUMN `is_hidden` TINYINT NOT NULL DEFAULT 0;
```

Comment:
These should go to an upgrade SQL script as mentioned before and just referenced here.